lib/http/timeout/global.rb

```diff
@@ -119,10 +119,10 @@ def reset_timer
         @started = Time.now
       end
 
-      def log_time
+      def log_time(klass)
```

Comment:
Because this is a private method, I'm assuming it was ok to modify its method signature. That was the most straightforward way I could see to handle all cases.